components/job-orchestration/job_orchestration/retention/search_results_handler.py

```
@@ -0,0 +1,66 @@
+
+logger = get_logger(SEARCH_RESULTS_RETENTION_HANDLER_NAME)
+
+
+def _get_latest_doc_timestamp(collection: pymongo.collection.Collection) -> int:
+    latest_doc = collection.find_one(sort=[(MONGODB_ID_KEY, pymongo.DESCENDING)])
+    if latest_doc:
+        object_id = latest_doc[MONGODB_ID_KEY]
+        if isinstance(object_id, ObjectId):
+            return int(object_id.generation_time.timestamp())
+        else:
+            raise ValueError(f"{object_id} is not a ObjectID")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)